x/staking/keeper/keeper.go

```diff
@@ -51,6 +51,7 @@ type Keeper struct {
 	RedelegationsByValDst         collections.Map[collections.Triple[[]byte, []byte, []byte], []byte]
 	RedelegationsByValSrc         collections.Map[collections.Triple[[]byte, []byte, []byte], []byte]
 	UnbondingDelegationByValIndex collections.Map[collections.Pair[[]byte, []byte], []byte]
+	LastValidatorPower            collections.Map[[]byte, []byte]
```

Comment:
I have to say that I would find useful if we can add comments next to the field précising what is expected as key and as value. Right now, it is really not clear that we expect until I look deeper in the code.
I believe this would be a good practice to start doing when we simply use byte slices.